.gitmodules

```diff
@@ -1,6 +1,7 @@
 [submodule "hyprland-protocols"]
 	path = subprojects/hyprland-protocols
-	url = https://github.com/hyprwm/hyprland-protocols
+	url = https://github.com/3l0w/hyprland-protocols
+	branch = feat/input-capture-impl
```

Comment:
marking this one as a blocker until this is removed so we dont forget (don't resolve)